### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by logan-anderson**
fixes issue: https://discord.com/channels/835168149439643678/835172431836479508/1035647160898826401

### CODE REVIEW

File: packages/@tinacms/cli/src/cmds/compile/index.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export const resetGeneratedFolder = async ({
   usingTs: boolean
   isBuild: boolean
 }) => {
+  console.log('called resetGeneratedFolder')
```
Comment (jeffsee55) [inline packages/@tinacms/cli/src/cmds/compile/index.ts:44]:
```suggestion
```

File: packages/@tinacms/cli/src/cmds/compile/index.ts [hunk 2]
```diff
@@ -63,19 +64,21 @@ export const resetGeneratedFolder = async ({
...
     console.log(e)
   }
+
   await fs.mkdirp(tinaGeneratedPath)
   const ext = usingTs ? 'ts' : 'js'
 
   // temp types file to allows the client to build
-  if (await !fs.pathExists(path.join(tinaGeneratedPath, `types.${ext}`))) {
+  if (!(await fs.pathExists(path.join(tinaGeneratedPath, `types.${ext}`)))) {
+    console.log('creating types file')
```
Comment (jeffsee55) [inline packages/@tinacms/cli/src/cmds/compile/index.ts:73]:
```suggestion
```

**Review by jeffsee55 — COMMENTED** [review]
Just some console.logs, looks good otherwise

**Comment by logan-anderson** [comment]
> Just some console.logs, looks good otherwise

fixed

**Pull request merged**